apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,134 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    setup(function() {
+      //this.sinon.useFakeTimers();
```

Comment:
You can remove the whole setup if you don't need it. BTW why not use `this.sinon.useFakeTimers()` here and `this.sinon.clock.tick()` in the test?